### DISCUSSION TITLE
The UVICORN dont start correctly.

### DISCUSSION THREAD

**Discussion by KUASSIISRAEL** [Kludex/uvicorn#1766] [Q&A] [answered]
When i start my WEB SERVER UVICORN version 0.19.0 i got this error:

** module 'uvloop' has no attribute 'EventLoopPolicy'**
<img width="1207" alt="Capture d’écran 2022-11-13 à 14 24 40" src="https://user-images.githubusercontent.com/64444325/201526835-4a78ee31-aaed-4849-9b50-3d0dbc090d8b.png">

**Comment by iudeen** [comment]
> The starting point for issues should usually be a discussion...
> 
> https://github.com/encode/uvicorn/discussions
> 
> Possible bugs may be raised as a "Potential Issue" discussion, feature requests may be raised as an "Ideas" discussion. We can then determine if the discussion needs to be escalated into an "Issue" or not.
> 
> This will help us ensure that the "Issues" list properly reflects ongoing or needed work on the project.
> 

Can you please open a [discussion](https://github.com/encode/uvicorn/discussions)?

**Comment by Kludex** [comment]
> > The starting point for issues should usually be a discussion...
> > [encode/uvicorn/discussions](https://github.com/encode/uvicorn/discussions)
> > Possible bugs may be raised as a "Potential Issue" discussion, feature requests may be raised as an "Ideas" discussion. We can then determine if the discussion needs to be escalated into an "Issue" or not.
> > This will help us ensure that the "Issues" list properly reflects ongoing or needed work on the project.
> 
> Can you please open a [discussion](https://github.com/encode/uvicorn/discussions)?

There's no need to suggest that, there's a button on the right to do that "Convert to discussion":
![image](https://user-images.githubusercontent.com/7353520/201609675-7d53e35e-6100-425b-b1b7-9adc20cb7e85.png)

**Reply by iudeen** [reply]
Okay, thanks!

**Comment by iudeen** [comment]
Is this a scenario similar to [this](https://stackoverflow.com/questions/58195742/uvloop-has-no-attribute-eventlooppolicy)?

**Reply by KUASSIISRAEL** [reply]
No. 

The issu which i have on the console is:

THE UVLOOP DONT HAVE THE ATTRIBUTE EventLoopPolicy

**Reply by iudeen** [reply]
Can you specify the version and if possible reinstall the latest Uvicorn by using `pip install uvicorn[standard] --upgrade`

**Comment by Kludex** [comment, marked as answer]
Can you provide a minimal, reproduce, example that I can run, or is this already solved?
